### PR TITLE
Implement API for Miners to expose contents in bulk to validators.

### PR DESCRIPTION
**Pull request by Sid-Data-Universe**

### CODE REVIEW

**Review by RusticLuftig — APPROVED** [review]

File: mydb
[the remark this answers is not in the document]
Comment (Sid-Data-Universe) [reply mydb:1]: You know we had one for myDb not mydb... fixed.

File: neurons/miner.py [hunk 1]
```diff
@@ -423,8 +465,10 @@ def default_blacklist(self, synapse: bt.Synapse) -> typing.Tuple[bool, str]:
             self.requests_by_hotkey[synapse.dendrite.hotkey] += 1
 
             # Blacklist if over request limit.
-            # We allow up to 4 requests in case a validator restarts and sends two pairs of index/bucket requests.
-            if self.requests_by_hotkey[synapse.dendrite.hotkey] > 4:
+            # We allow up to 8 requests.
+            # 4 to allow for 1 get index, 1 get data entity bucket, and 2 get obfuscated data entity buckets.
+            # Then we double that to allow for the case of a validator restarting.
```
[the remark this answers is not in the document]
Comment (Sid-Data-Universe) [reply neurons/miner.py]: Done. Went with 5 * 2 for now.

File: neurons/miner.py [hunk 2]
```diff
@@ -368,51 +373,113 @@ async def get_data_entity_bucket_priority(
...
     def default_blacklist(self, synapse: bt.Synapse) -> typing.Tuple[bool, str]:
         """The default blacklist that only allows requests from validators."""
-        if synapse.dendrite.hotkey in [
-            "5Gpt8XWFTXmKrRF1qaxcBQLvnPLpKi6Pt2XC4vVQR7gqNKtU"
-        ]:
+        hotkey = synapse.dendrite.hotkey
+        ip = synapse.dendrite.ip
+        synapse_type = type(synapse)
+
+        if hotkey in ["5Gpt8XWFTXmKrRF1qaxcBQLvnPLpKi6Pt2XC4vVQR7gqNKtU"]:
```
Comment (RusticLuftig) [inline neurons/miner.py:431]: Note for future: We should remove this

**Review by RusticLuftig — APPROVED** [review]

File: neurons/miner.py (hunk 2 above)
Comment (Sid-Data-Universe) [reply neurons/miner.py]: True, went ahead and did that now with the other blacklist changes.

**Review by RusticLuftig — APPROVED** [review]

**Pull request merged**